docs/docs/inputs_outputs.md

```diff
@@ -7,40 +7,65 @@ This section covers what data you can pass to zamba and what the output looks li
 Using data collected and annotated by partners at [The Max Planck Institute for
 Evolutionary Anthropology](https://www.eva.mpg.de/index.html) and [Chimp &
 See](https://www.chimpandsee.org/), the prepackaged model in `zamba` was
-trained to detect 23 species of animal––as well as `blank` videos (no animal
+trained to detect 31 species of animal––as well as `blank` videos (no animal
 present).
 
 The possible class labels in `zamba`'s default model are:
 
+* `aardvark`
+* `antelope_duiker`
+* `badger`
+* `bat`
 * `bird`
 * `blank`
 * `cattle`
-* `chimpanzee`
-* `duiker`
+* `cheetah`
+* `chimpanzee_bonobo`
+* `civet_genet`
 * `elephant`
-* `forest buffalo`
+* `equid`
+* `forest_buffalo`
+* `fox`
+* `giraffe`
 * `gorilla`
+* `hare_rabbit`
 * `hippopotamus`
 * `hog`
 * `human`
 * `hyena`
-* `large ungulate`
+* `large_flightless_bird`
 * `leopard`
 * `lion`
-* `other (non-primate)`
-* `other (primate)`
+* `mongoose`
+* `monkey_prosimian`
 * `pangolin`
 * `porcupine`
 * `reptile`
 * `rodent`
-* `small antelope`
-* `small cat`
-* `wild dog`
+* `small_cat`
+* `wild_dog_jackal`
+
+<!-- TODO: add details of european model. who collected and labels the data?><!-->
+
+*New in Zamba v2:* Thanks to new data shared by [<SOURCE>], `zamba` now also includes a prepackaged model to identify 11 common European species. The class labels in `zamba`'s European model are:
```

Comment:
Where did we get the data for the european model? Who should we credit for collection / annotation?

Reply:
I forwarded the metadata we have. For all the models, we should draft descriptions based on what you can put together and then we can ask our collaborators to confirm the details